test/ssh_command_test.rb

```diff
@@ -69,7 +75,7 @@ def test_try_connection_with_gateway_connects_using_ssh_gateway
     cmd = command("10.0.0.1", "--ssh-gateway=user@gateway")
     ssh_mock = mock 'ssh_mock'
     Net::SSH::Gateway.expects(:new).with('gateway', 'user').returns(ssh_mock)
-    ssh_mock.expects(:ssh).with(cmd.host, cmd.user, cmd.connection_options.except(:gateway))
+    ssh_mock.expects(:ssh).with(cmd.host, cmd.user, cmd.connection_options.select{|o| o != :gateway})
```

Comment:
I think hash.except is something in rails? Does ruby support that itself?

I got this when I ran the test:
>SshCommandTest#test_try_connection_with_gateway_connects_using_ssh_gateway:
NoMethodError: undefined method `except' for #<Hash:0x00000601f0d7d0>